src/governance/governance.cpp

```diff
@@ -516,8 +516,7 @@ std::vector<const CGovernanceObject*> CGovernanceManager::GetAllNewerThan(int64_
         }
 
         // ADD GOVERNANCE OBJECT TO LIST
-        const CGovernanceObject* pGovObj = &(objPair.second);
-        vGovObjs.push_back(pGovObj);
+        vGovObjs.push_back(std::make_shared<const CGovernanceObject>(objPair.second));
```

Comment:
This is a bad idea anyway. If I'm understanding correctly, right here you're creating a copy, creating a shared_ptr to that copy, and returning the vector of those shared_ptrs. You could simply return a vector of copies if you don't like my lock extension idea